components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/MenuButton.kt

```diff
@@ -96,6 +96,10 @@ internal class MenuButton(
         }
     }
 
+    fun dismissMenu() {
+        menu?.dismiss()
```

Comment:
@mauryat  To test this you could use a similar approach as before mock `menu` and verify that every time that `MenuButton#dismissMenu()` is called `menu?.dismiss()` is called too :)